<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BackPic</class>
 <widget class="QDialog" name="BackPic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>211</width>
     <height>221</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>188</width>
      <height>219</height>
     </rect>
    </property>
    <widget class="QToolButton" name="btn1">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>10</y>
       <width>140</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 0, 0, 200);
color: rgba(85, 255, 96, 255);
font: 12pt &quot;Arial&quot;;
border-radius:12px;</string>
     </property>
     <property name="text">
      <string>绿野仙踪</string>
     </property>
    </widget>
    <widget class="QToolButton" name="btn2">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>50</y>
       <width>140</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 0, 0, 200);
color: rgba(85, 255, 96, 255);
font: 12pt &quot;Arial&quot;;
border-radius:12px;</string>
     </property>
     <property name="text">
      <string>幽夜暗香</string>
     </property>
    </widget>
    <widget class="QToolButton" name="btn3">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>90</y>
       <width>140</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 0, 0, 200);
color: rgba(85, 255, 96, 255);
font: 12pt &quot;Arial&quot;;
border-radius:12px;</string>
     </property>
     <property name="text">
      <string>夏日心情</string>
     </property>
    </widget>
    <widget class="QToolButton" name="btn4">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>130</y>
       <width>140</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 0, 0, 200);
color: rgba(85, 255, 96, 255);
font: 12pt &quot;Arial&quot;;
border-radius:12px;</string>
     </property>
     <property name="text">
      <string>美女凝视</string>
     </property>
    </widget>
    <widget class="QToolButton" name="btn5">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>170</y>
       <width>140</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 0, 0, 200);
color: rgba(85, 255, 96, 255);
font: 12pt &quot;Arial&quot;;
border-radius:12px;</string>
     </property>
     <property name="text">
      <string>可爱动人</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="close_btn">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>0</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/image/close24.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">
color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>列表透明度</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="horizontalSlider">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="song_image.qrc"/>
 </resources>
 <connections/>
</ui>
